.github/workflows/gpu_tests.yaml

```diff
@@ -53,6 +53,8 @@ jobs:
         run: torchrun --nproc-per-node 4 test/test_autosplit.py
       - name: Test skip connection support
         run: torchrun --nproc-per-node 4 test/test_skip_conn.py
+      - name: Test Composability
+        run: python test/test_composability.py
```

Comment:
looks like the job also needs to `pip install expecttest` since the pt distributed tests use it